sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -841,11 +841,12 @@ def run_Read(self, transform_node):
                           transform.source.id_label)
       if transform.source.with_attributes:
         # Setting this property signals Dataflow runner to return full
-        # PubsubMessages instead of just the payload.
+        # PubsubMessages instead of just data.
```

Comment:
"the payload data" for clarity?  This is the official doc for PubsubMessage: https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage

Reply:
Fixed